js/binance.js

```diff
@@ -2062,16 +2062,19 @@ module.exports = class binance extends Exchange {
 
     async fetchStatus (params = {}) {
         const response = await this.sapiGetSystemStatus (params);
-        let status = this.safeString (response, 'status');
-        if (status !== undefined) {
-            status = (status === '0') ? 'ok' : 'maintenance';
-            this.status = this.extend (this.status, {
-                'status': status,
-                'updated': this.milliseconds (),
-                'info': response,
-            });
-        }
-        return this.status;
+        //
+        //     {
+        //         "status": 0,              // 0: normal，1：system maintenance
+        //         "msg": "normal"           // "normal", "system_maintenance"
+        //     }
+        //
+        const statusRaw = this.safeString (response, 'status');
+        return {
+            'status': this.safeString ({ '0': 'ok', '1': 'maintenance' }, statusRaw, statusRaw),
```

Comment:
I would suggest to move this parsing bit out to a method like `parseExchangeStatus` or just `parseStatus`, what do you think ?

Reply:
sure, I'll do that. 
imho, even though 'parseStatus' is more suited to for fetchStatus method, still, it shouldn't be `parseStatus` as we might reserve that method name for future usage in base (if we will have some unified mappings for parsers, then we might need to have `parseStatus` shorthand method, idk), so maybe `parseExchangeStatus` could be used.

Reply:
side-question: if we move out parsing from there, them maybe whole `parseExchangeStatus` should be moved into separate parser for whole response object, as regular parsers do? i just think it might not be needed for fetchStatus method, but idk, as you tell.

Reply:
I was just thinking of splitting that long line, cause we might have more statuses in the other exchanges. But it's ok, we can keep it in-place.

Reply:
so, it would be like: 
```
fetchStatus(){
    response = await..
    return this.parseExchangeStatus (response);
}
...
parseExchangeStatus (data) {
   return {
     ...
     'status': this.parseExchangeStatusValue ( rawStatus ); 
     ...
   }
}

parseExchangeStatusValue (raw){
   ..
}
```

this is like other complete parsers.

Reply:
maybe , at this specific moment, lets don't do that parser chains and leave fetchStatus self-contained?